packages/alba/alba.0.4.1/opam

```diff
@@ -0,0 +1,29 @@
+opam-version: "2.0"
+synopsis: "Alba compiler"
+maintainer: "Helmut Brandl <helmut.brandl@gmx.net>"
+authors: "Helmut Brandl <helmut.brandl@gmx.net>"
+homepage: "http://albatross-lang.sourceforge.net"
+license: "GPLv2"
+dev-repo: "git+https://github.com/hbr/albatross.git"
+bug-reports: "http://github.com/hbr/albatross"
+build: [
+  [make "test_alba"] {with-test}
```

Comment:
This call contains call to dune without the `-p` or `--profile=release` argument. This is not future proof as dev arguments can change with new versions of OCaml or new versions of dune. Couldn't this be done using the `dune runtest` command instead? Looking from the Makefile this seems doable.

If you don't have time to integrate it in dune, I'd suggest to remove this line instead.